test/test_contra_env_setup.py

```diff
@@ -49,25 +49,37 @@ def setUp(self):
 
 
     def test(self):
+        distro = self.params.get('distro', default='Fedora')
+
         # Set the SSH options and port
         env = {'ANSIBLE_CONFIG': 'test/ansible.cfg'}
         ssh_args = ['-o UserKnownHostsFile=/dev/null',
                     '-o StrictHostKeyChecking=no']
 
-        # Install python and update nettle to libvirt network
-        cmd = (' ansible localhost -i "localhost," --become '
-               ' -m raw -a "dnf install -y python libselinux-python git; '
-               ' dnf update -y nettle" --user %s --ssh-common-args="%s" ' %
-               (self.vm.username, ' '.join(ssh_args)))
-        process.run(cmd, env=env)
-
+        # Install python, git and update nettle to libvirt network
+        if distro == 'Fedora':
```

Comment:
Seems like the only difference in these 2 if statements is "yum" and "dnf". Maybe simplify to set a var for "yum" or "dnf" and only have the one cmd?